tests/tests.go

```diff
@@ -351,11 +330,17 @@ func changeManagerReplicas(numOfReplica int32) error {
 			return false
 		}
 
+		numberOfReadyPods := int32(0)
 		for _, podObject := range podsList.Items {
-			if podObject.Status.Phase != corev1.PodRunning {
-				return false
+			for _, condition := range podObject.Status.Conditions {
+				if condition.Type == corev1.PodReady && condition.Status == corev1.ConditionTrue {
+					numberOfReadyPods += 1
+				}
 			}
 		}
+		if numberOfReadyPods < numOfReplica {
+			return false
+		}
```

Comment:
As @ormergi point out, just checking `managerDeployment.Status.ReadyReplicas != numOfReplica` is enough since it's more or less the same as this check.